docs/manifest-types.md

```diff
@@ -29,6 +29,23 @@ All manifests have such structure for common types. More data you can find below
 
 ## Available manifest types
 
+| Type                                                  | Generation | Integration |
+|-------------------------------------------------------|------------|-------------|
+| [wire-plugin](#wire-plugin)                           | YES        | YES         |
```

Comment:
```suggestion
| [wire-plugin](#wire-plugin-manifest)                           | YES        | YES         |
```

I think the keys must be the slug of the title, unfortunately, for the links to work

Reply:
We just have to use a full slug, for example `wire-plugin-manifest`